### PR TITLE
data: Ankur6702

### PR DESCRIPTION
**Pull request by Ankur6702**

### CODE REVIEW

**Review by Tejender1521 — APPROVED** [review]
Everything Looks Good!!!

File: data/Ankur6702.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
+  "name": "Ankur Agarwal",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "MERN Stack Developer | Google DSC Lead | Student Placement Coordinator | IIIT Kota",
+  "avatar": "https://drive.google.com/file/d/1fqSNRfbD8j9xUNJ7t-1ZdDeEwyIgcM6n/view?usp=sharing",
```
Comment (kumarsonsoff3) [inline data/Ankur6702.json:6]: Please fix your avatar link to a more dynamic URL (avatar will get updated automatically when you change your GitHub Profile) + it's too short and clean! 😄

```suggestion
  "avatar": "https://github.com/Ankur6702.png",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by Tejender1521 — APPROVED** [review]
Now Everything looks good and ready to merge....

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge, Thanks for your Contribution! 🎉 

<p align="center"> </p>

**Pull request merged**